main.go

```diff
@@ -80,7 +80,7 @@ func probeHandler(w http.ResponseWriter, r *http.Request, c *config.Config, logg
 	module, ok := c.Modules[moduleName]
 	if !ok {
 		http.Error(w, fmt.Sprintf("Unknown module %q", moduleName), http.StatusBadRequest)
-		level.Debug(logger).Log("msg", "Error ", "err", "unknown module name", "module", moduleName)
+		level.Debug(logger).Log("msg", "Error finding module", "module", moduleName)
```

Comment:
It's not about finding, keep it consistent with the error we send back over HTTP - so Unknown module.

Reply:
Done, just in time for the Toy Show 🇮🇪 😄